tests/Moq.Tests/AwaitOperatorFixture.cs

```diff
@@ -589,6 +590,29 @@ public async Task Mock_Of()
 			Assert.Equal(expectedSameName, actualName);
 		}
 
+		[Fact]
+		public void Await__is_represented__in_setup_Expression()
+		{
+			var mock = new Mock<IPerson>();
+			mock.Setup(m => Await(m.DoSomethingTaskAsync()));
+			var setup = mock.Setups.Last();
+			Assert.Contains("aWAiT", setup.Expression.ToStringFixed(), StringComparison.OrdinalIgnoreCase);
+			//               ^^^^^
+			// NOTE: Depending on the exact scenario, the `Await` method name may get converted to
+			// a faux `await` keyword (e.g. with `SetupSet`). Right now, we don't care too much about
+			// that slight loss of accuracy, as long as a clue for the occurred await remains.
```

Comment:
Perhaps, for overall consistency, all setup expressions should be post-processed to rewrite `Await` calls as `(await …)`. That might not be worth the trouble if we later decide to go in the oppsosite direction (i.e. if we want to try and preserve the original `Await` method identity in expression trees reconstructed from delegates).